src/ch/imedias/rsccfx/view/RsccEnterTokenPresenter.java

```diff
@@ -28,20 +38,28 @@ private void attachEvents() {
       view.isValidimg.setImage(new Image(validationImage(view.tokentxt.getText())));
     });
 
+    view.connectbtn.setOnAction(
+        event -> {
+          model.keyProperty().set(view.tokentxt.getText());
+          model.connectToUser(model.getKey(),FORWARDING_PORT,KEY_SERVER_IP,
+              KEY_SERVER_HTTP_PORT);
+        }
+    );
+
   }
 
   /**
    * Validates the token and displays a symbolic image.
    */
-  public String validationImage(String token) {
-
+  public String validationImage(String token) { // FIXME: method names should be formed as "verb + noun", so here it would be "validateImage"
```

Comment:
@Kennox method names should be formed as "verb + noun", so here it would be "validateImage"